api-remove.go

```diff
@@ -183,32 +188,6 @@ func (c Client) RemoveObjectsWithContext(ctx context.Context, bucketName string,
 	return errorCh
 }
 
-// RemoveObjectsWithOptionsContext - Identical to RemoveObjects call, but accepts context to
-// facilitate request cancellation and options to bypass governance retention
-func (c Client) RemoveObjectsWithOptionsContext(ctx context.Context, bucketName string, objectsCh <-chan string, opts RemoveObjectsOptions) <-chan RemoveObjectError {
```

Comment:
Let's keep this name itself, otherwise this is a breaking change

Reply:
ok